terraform/development/main.tf

```diff
@@ -142,5 +142,12 @@ data "aws_iam_policy_document" "sns_topic_policy" {
 
       sid = "dev-statement"
     }	
-}	
+}
+
+# TO DO: Add lambda function name once created
+# resource "aws_lambda_event_source_mapping" "aws_lambda_event_source" {
+#   event_source_arn  = aws_dynamodb_table.TenureInformation.stream_arn
+#   function_name     = aws_lambda_function.lambda_function.arn
+#   starting_position = "LATEST"
+# }
```

Comment:
Could add a simple lambda deployed as part of the TenureInformation API.

(add a simple lambda that logs "hello world" to Tenure Information APIs serverless.yaml and then use its ARN here)

Would be nice to have the lambda as part of this PR sort of as a proof of concept that we could check in the cloudwatch logs. :thinking: 

Idk, what do you think?